barretenberg/ts/src/bb_backends/wasm.ts

```diff
@@ -76,7 +75,7 @@ export class BarretenbergWasmAsyncBackend implements IMsgpackBackendAsync {
       await wasm.init(
         module,
         threads,
-        proxy(options.logger ?? createDebugLogger('bb_wasm_async')),
```

Comment:
in my original vision, users providing a logging function would have access to this string. but this is fine